doc/JSON_FLAGS.md

```diff
@@ -243,6 +244,7 @@ Flags used to describe monsters and define their properties and abilities.
 - ```DIGS``` Digs through the ground.
 - ```DOGFOOD``` Becomes friendly / tamed with dog food.
 - ```DRIPS_NAPALM``` Occasionally drips napalm on move.
+- ```DRIPS_GASLOLINE``` Occasionally drips gasoline on move.
```

Comment:
```suggestion
- ```DRIPS_GASOLINE``` Occasionally drips gasoline on move.
```

Reply:
I Really do lack quality control dont I.
however, gasloline is a good typo.